cat-launcher/src/pages/ToolsPage/components/ImportGameData.tsx

```
@@ -0,0 +1,93 @@
+        <h2 className="text-2xl font-bold">Import Game Data</h2>
+        <p className="text-muted-foreground">
+          Import game data from a zip archive or a folder. A backup
+          will be created before importing.
+        </p>
+      </div>
+
+      <div className="space-y-4 max-w-md">
+        <div className="space-y-2">
+          <label className="text-sm font-medium">
+            Select Game Variant
+          </label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)